lib/content-services/src/lib/search/services/base-query-builder.service.ts

```diff
@@ -615,4 +633,16 @@ export abstract class BaseQueryBuilderService {
             queryParamsHandling: 'merge'
         });
     }
+
+    private readonly queryFragmentsHandler: ProxyHandler<{ [key: string]: any }> = {
```

Comment:
Similar comment for usages for any in that function: 
https://github.com/Alfresco/alfresco-ng2-components/pull/11386/files#r2567686029